CHANGELOG.md

```diff
@@ -33,6 +33,7 @@ Bug Fixes:
 - Ensure that we support ${workspaceFolder} when initializing cmake information. [#3658](https://github.com/microsoft/vscode-cmake-tools/issues/3658)
 - Fix issue where correcting `cmake.cmakePath` is still broken. [#3570](https://github.com/microsoft/vscode-cmake-tools/issues/3570)
 - Fix CMakePresets.json schema validation. [#3651](https://github.com/microsoft/vscode-cmake-tools/issues/3651)
+- Add support for vscode variable in debugConfig environment. [#3711](https://github.com/microsoft/vscode-cmake-tools/issues/3711
```

Comment:
Suggestion:  "Add support for variable expansion in \`debugConfig.environment\`"

Reply:
OK